gossip/minimal/goshawk_test.go

```diff
@@ -136,11 +137,14 @@ func TestValidateSTH(t *testing.T) {
 	if err != nil {
 		t.Fatalf("failed to create client: %v", err)
 	}
-	o := originLog{logConfig: logConfig{
-		Name: "argon2018",
-		URL:  "http://ct.googleapis.com/logs/argon2018",
-		Log:  client,
-	}}
+	o := originLog{
+		reporter: &incident.LoggingReporter{},
```

Comment:
Given that this is the default when you leave `reporter` nil, why not omit it?

Reply:
It's only the default for creating a `NewGoshawk`; because the unit test uses the internal `originLog` it needs to be set manually.